recon/src/recon/mod.rs

```diff
@@ -218,9 +252,9 @@ impl<H: Hash> Message<H> {
         right_fencepost: &str,
         received_hash: &H,
         local_keys: &Recon,
-    ) {
+    ) -> bool {
```

Comment:
todo: add doc string here explain that the return value is whether the sets are reconciled